src/GAP.jl

```diff
@@ -318,6 +320,28 @@ function randseed!(seed::Union{Integer,Nothing}=nothing)
     nothing
 end
 
+"""
+    versioninfo(io::IO = stdout; jll::Bool = false)
+
+Print the version numbers of GAP.jl and GAP,
+and version numbers and installation paths of all currently loaded GAP packages.
+Note that these paths can be nonstandard because Julia's package manager
+does not control which available version of a GAP package gets loaded.
+
+If `jll` is `true` then also the underlying binary packages (jll),
+if available, of all installed (not necessarily loaded) packages
+are included in the output.
+"""
+function versioninfo(io::IO = stdout; jll::Bool = false, full::Bool = false, padding::String = "")
+  if full
+    jll = true
+  end
+  # We cannot use `Pkg.dependencies()` because (depending on the project)
+  # GAP is perhaps not contained.
+  println(io, "GAP.jl version ", Compat.pkgversion(@__MODULE__))
```

Comment:
This needs Compat `4.11` or later, but the current compat of Compat.jl is `4.4.0`

Reply:
o.k.